lib/Rex/Config.pm

```diff
@@ -1739,7 +1739,11 @@ sub read_config_file {
   $config_file ||= _home_dir() . "/.rex/config.yml";
 
   if ( -f $config_file ) {
-    my $yaml = eval { local ( @ARGV, $/ ) = ($config_file); <>; };
+    my $yaml = do {
+      use IO::File;
+      my $fh = IO::File->new( $config_file, 'r' );
+      join '', $fh->getlines();
+    };
```

Comment:
What do you think about using `LoadFile` from the YAML module to read this YAML file below instead of reading the whole file and `eval`ing it?

Reply:
done